meeshkan/serve/mock/faker/__init__.py

```diff
@@ -0,0 +1,9 @@
+from meeshkan.serve.mock.faker.schema_faker import MeeshkanSchemaFaker
+
+try:
+    import spacy
```

Comment:
I thought the idea was to put spacy stuff to a different (optional) `meeshkan-nlp` package?

Reply:
Although if we have `pip install meeshkan[x]` or whatever then this is ok because it is wrapped in the `try` bloc.

Reply:
Also, how would spacy effect the faker? Isn't spacy what's used in the builder stage?

Reply:
Yes, right now we may use both fakers without spacy. Spacy is needed only in the build step. And for old-style specs, the new faker behaves in the same way as the old one.